src/featureFlags/flags/permit2.ts

```diff
@@ -0,0 +1,7 @@
+import { BaseVariant, FeatureFlag, useBaseFlag } from '../index'
```

Comment:
i want to refactor this system to just have a single hook "useFeatureFlags" instead of having these files eventually